webapp-django/crashstats/crashstats/management/commands/archivescraper.py

```diff
@@ -64,24 +64,16 @@
 import copy
 from functools import partial
 import json
-import logging
 
-from configman import Namespace, class_converter
 import more_itertools
-import psycopg2
+from pyquery import PyQuery as pq
 from six.moves.urllib.parse import urljoin
 
-from socorro.cron.base import BaseCronApp
-from socorro.lib.requestslib import session_with_retries
-from socorro.lib.transaction import transaction_context
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
 
-# NOTE(willkg): We have to do this because lxml 4.2.5 has imports that kick up
-# warnings in Python 3. Once lxml puts out a new version with a fix, we can
-# stop doing this.
```

Comment:
We've upgraded lxml since I wrote this note and this warnings thingamabob, so it's not relevant anymore and we can remove it.